jetty-core/jetty-coreapp/src/main/java/org/eclipse/jetty/coreapp/CoreAppContext.java

```diff
@@ -66,9 +66,9 @@
  *     unpacked into the temp directory defined by this core webapp.
  * </p>
  */
-public class CoreContextHandler extends ContextHandler implements Deployable
+public class CoreAppContext extends ContextHandler implements Deployable
```

Comment:
I think the javadoc could be tweaked a little bit.  Something like:
```
/**
 * A specialised ContextHandler that deploys a Web Application based on Jetty core APIs.
```

I then think all the text about `metadata` is kind of meaning less without a bit more of an introduction as to the layout of a coreapp